src/generated/resources/assets/aether_genesis/lang/en_us.json

```diff
@@ -15,6 +15,7 @@
   "block.aether_genesis.storm_aercloud": "Storm Aercloud",
   "block.aether_genesis.stripped_skyroot_log_wall": "Stripped Skyroot Log Wall",
   "block.aether_genesis.stripped_skyroot_wood_wall": "Stripped Skyroot Wood Wall",
+  "config.aether.client.client.night_music_tracks": "Adds some nice night tracks to the Aether's music selection. Also disables the default music manager for the Aether, to preveent overlap",
```

Comment:
The typo is still there.

Reply:
uhh, that’s odd, I’ll try to fix it at some point soon-ish

Reply:
oh wait I forgot to rerun the datagen, so that’s why it’s still there

Reply:
Actually wait, I can just edit in the browser. It should be fixed now, hopefully